cmd/kubeadm/app/phases/addons/manifests.go

```diff
@@ -71,9 +71,8 @@ spec:
       - name: kube-proxy
         image: {{ .Image }}
         imagePullPolicy: IfNotPresent
-        # TODO: This is gonna work with hyperkube v1.6.0-alpha.2+: https://github.com/kubernetes/kubernetes/pull/41017
         command:
-        - kube-proxy
+        - /usr/local/bin/kube-proxy
```

Comment:
Ok, this came at the same time when cherry-picking, but it's okay.
Basically, there is a chance that we won't have shells in the future, so we should point at the absolute path (which now is the same for kube-* images and hyperkube thanks to a PR I made last week)

cc @timstclair and ref: https://github.com/kubernetes/kubernetes/issues/40248